definitions/contested/json/CaseField/CaseField-nonprod.json

```diff
@@ -6,5 +6,23 @@
     "Label": "D11 document",
     "FieldType": "Document",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "latestDraftDirectionOrder",
```

Comment:
same as above, don't use generic nonprod files. I think your ticket should be toggled on.